engine/src/main/java/org/terasology/engine/physics/bullet/BulletPhysics.java

```diff
@@ -140,6 +163,52 @@ public btDiscreteDynamicsWorld getWorld() {
     }
     //*****************Physics Interface methods******************\\
 
+
+    private int addRenderBound(StandardMeshData meshData, AABBf bounds, int index) {
```

Comment:
Is this what you added as part of that other PR, or is this something different?

Reply:
originally did the bounded box rendering stuff here.